src/test/ui/associated-consts/assoc-const.rs

```diff
@@ -0,0 +1,21 @@
+pub trait Foo {
```

Comment:
you can enable the feature in this test so it actually tests that everything works